modular_skyrat/modules/moretraitoritems/code/modules/clothing/suits/armor.dm

```diff
@@ -0,0 +1,11 @@
+//obj/item/clothing/suit/armor/legate
+	//icon = 'modular_skyrat/modules/customization/icons/obj/clothing/suits.dmi'
+	//worn_icon = 'modular_skyrat/modules/customization/icons/mob/clothing/suit.dmi'
+	//name = "ninth legion legate armor"
+	//desc = "Glistening gold armor, unlike most of the modern Ninth's armor, this is authentic, straight from the 130s."
+	//icon_state = "leglegate"
+	//body_parts_covered = CHEST|ARMS
+	//armor = list(MELEE = 45, BULLET = 20, LASER = 0,ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 10, ACID = 20, WOUND = 10)
+	//mutant_variants = NONE
```

Comment:
Why is this in this PR? Update the PR to reflect it or remove this

Reply:
yeah i'm removing em